api/types/desktop.go

```diff
@@ -218,3 +220,84 @@ func (f *WindowsDesktopFilter) Match(req WindowsDesktop) bool {
 	}
 	return true
 }
+
+// WindowsDesktops represents a list of windows desktops.
+type WindowsDesktops []WindowsDesktop
+
+// Len returns the slice length.
+func (s WindowsDesktops) Len() int { return len(s) }
+
+// Less compares desktops by name and host ID.
+func (s WindowsDesktops) Less(i, j int) bool {
+	return s[i].GetName() < s[j].GetName() && s[i].GetHostID() < s[j].GetHostID()
```

Comment:
This doesn't seem correct.

I think you want to sort by name, then host ID, so something like this:

```
switch {
case s[i].GetName() < s[j].GetName():
    return true
case s[i].GetName() > s[j].GetName():
    return false
default:
    return s[i].GetHostID() < s[j].GetHostID()  
}
```

Reply:
gah, didn't realize it was incorrect this entire time, sorry

Reply:
No worries! It still would have sorted them in _some_ order, probably just not the order you'd expect :-)